.gitignore

```diff
@@ -4,3 +4,4 @@ _site
 .jekyll-metadata
 .bundle
 .idea/
+.DS_Store
```

Comment:
Nit: For files, we usually want to have them end with a single blank line - looks like a couple in this PR are missing that.